dapps/digital-twin-detail/src/components/DigitalTwinDetail.scss

```diff
@@ -79,3 +79,7 @@
     font-weight: bold;
   }
 }
+
+/deep/ .dropdown-toggle-no-caret::after {
```

Comment:
Can it be moved to the the ui.libs general style?

Reply:
I believe this is already resolved with the new bootstrap version, I'll try to remove it.